### PR TITLE
Feat/se3 create

### PR DESCRIPTION
**Pull request by justipaula**
senior eng 3 description

### CODE REVIEW

File: eng-paths/director_of_engineering.md
Comment (mariah-hammer) [inline eng-paths/director_of_engineering.md:1]: @justipaula why did you remove the VP of Engineering info from the director role?

File: eng-paths/sr_software_engineer_3.md [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+<tr>
+<td  markdown="1">
+
+2-3 Products / Key components
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+- Multiple Products/Key Components.
+- All team's domain and tasks that involve cross-teams work.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:76]: This wording is hard to interpret. Are you trying to say the domains of all teams? Or all domains of their core team?

File: eng-paths/sr_software_engineer_3.md [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Technical expert on multiple products / key components, knowing all the extent of the team's domain.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:109]: Similar comment here for the "knowing all the extent of the team's domain" portion of the sentence. Are you referring to all teams or the individual's core team? (if the former, then 'extent of all teams' domains' if the latter then 'extent of their team's domain')

File: eng-paths/sr_software_engineer_3.md [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+<tr><td>
+
+Sr Software Engineer 2
+
+</td></tr>
+<tr><td  markdown="1">
+
+- Technical expert on 2 products / key components, solid on the rest of the platform.
+- Expert in primary focus area (BE/FE/Data/or DevOps).
+- Strong proficiency in DevOps, PG/CH, Observability, and other platform technologies.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:99]: Does it make sense for this one to be at the Senior 3 level now? Allows more room for someone (like a Senior Front End engineer) to move up their path with their expertise in their primary focus area.

So maybe at the Senior 2 level we just drop the 'strong' and put 'Proficiency in...' and then the "Strong proficiency" is introduced for Senior 3?

File: eng-paths/sr_software_engineer_3.md [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Technical expert on multiple products / key components, knowing all the extent of the team's domain.
+- Expert in primary focus area (BE/FE/Data/or DevOps).
+- Strong proficiency in DevOps, PG/CH, Observability, and other platform technologies.
+- Thrives in an environment with ambiguous scope, clarifying uncertainties and defining clear paths forward with the best outcomes given the goals.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:112]: I really like this one!

File: eng-paths/sr_software_engineer_3.md [hunk 5]
```diff
@@ -0,0 +1,381 @@
...
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Takes proactive ownership of the team's technical scope and responsibilities, not only delivering on tasks, but also strategically establishing, creating buy-in and executing a short and long-term vision for the team's technical direction.
+- Anticipates future needs, articulates clear plans, and secure buy-in from key stakeholders and team members.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:164]: `secure` --> `secures`

File: eng-paths/sr_software_engineer_3.md [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Thrives in communication, ensuring clarity and objectiveness for individuals with varying levels of technical expertise.
+- Able to clearly articulate pros/cons, costs and tradeoffs of possible decisions.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:222]: What about `Able to clearly articulate pros and cons of possible options, including: costs, tradeoffs, risks, and benefits`?

File: eng-paths/sr_software_engineer_3.md [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+- Proactively establishes the team’s vision and strategic direction.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:223]: Should we say `technical vision` rather than just `vision`?

File: eng-paths/sr_software_engineer_3.md [hunk 8]
```diff
@@ -0,0 +1,381 @@
...
+- Inspires and motivates team members towards a shared vision, cultivating a culture of innovation and excellence.
+- Helps on the professional development of team members through mentorship, knowledge sharing, and promoting continuous learning .
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:225]: FYI there's an extra space at the end of this one

File: eng-paths/sr_software_engineer_3.md [hunk 9]
```diff
@@ -0,0 +1,381 @@
...
+- Holds the team accountable to high technical standards.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:226]: `Holds the team accountable to high technical standards and best practices`

File: eng-paths/sr_software_engineer_3.md [hunk 10]
```diff
@@ -0,0 +1,381 @@
...
+<td  markdown="1">
+
+- Deliver Results.
+- Service Oriented.
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+- Deliver results.
+- Stretegic thinking.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:278]: Typo in Strategic

File: eng-paths/sr_software_engineer_3.md [hunk 11]
```diff
@@ -0,0 +1,381 @@
...
+
+</td>
+<td  markdown="1">
+
+- Deliver results.
+- Stretegic thinking.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:284]: Typo in Strategic

File: eng-paths/sr_software_engineer_3.md [hunk 12]
```diff
@@ -0,0 +1,381 @@
...
+
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Has been a tech lead for 2+ year.
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:357]: `year` --> `years`

File: eng-paths/sr_software_engineer_3.md [hunk 13]
```diff
@@ -0,0 +1,381 @@
...
+- Has proposed and led 3+ large technical epics (from design to execution) .
```
Comment (mariah-hammer) [inline eng-paths/sr_software_engineer_3.md:358]: extra space at the end

File: eng-paths/director_of_engineering.md
Comment (justipaula) [reply eng-paths/director_of_engineering.md:1]: I haven't 👀 I guess that's something on the compiler's logic which I'm not aware of. Will give it a look

File: eng-paths/sr_software_engineer_3.md (hunk 1 above)
Comment (justipaula) [reply eng-paths/sr_software_engineer_3.md:76]: I'd say all domains of their core team + cross-teams tasks, lemme fix it

File: eng-paths/sr_software_engineer_3.md (hunk 11 above)
Comment (justipaula) [reply eng-paths/sr_software_engineer_3.md:284]: lol where did I get that from 🤦‍♀️🤣 thanks!

File: eng-paths/director_of_engineering.md
Comment (justipaula) [reply eng-paths/director_of_engineering.md:1]: @partner0 lol a bit lost here, where's the position being taken from? I didn't change anything on removing the VP from the Director description, but somewhere on the compiler it went away (it's not appearing on the render on eng-paths-generator 🤔 ). Could you shed some light if you remember where this is being taken from? thankss 🙏🏽

Comment (partner0) [reply eng-paths/director_of_engineering.md:1]: It was the range on the spreadsheet as you found 🙂

Comment (partner0) [reply eng-paths/director_of_engineering.md:1]: Line 44 should be Senior Staff in the index, currently you have Staff twice (one of them pointing to the right sr staff file though

**Comment by partner0** [comment]
I reviewed Sr SE 3 on [the spreadsheet](https://docs.google.com/spreadsheets/d/1PR2SKx3wIFfOOjx8XbzAGxlG4BmA3YWlhNol5HvxuTU/edit?gid=0#gid=0) rather than the PR, although I did look at the PR diagonally too. It all looks excellent to me. I am super happy you were able to find a middle ground between SE 2 and Staff, and in my view, it does set a high bar for our current Sr 2. Thank you for all the hard work on this one! 🙏

File: eng-paths/sr_staff_engineer.md [hunk 14]
```diff
@@ -85,9 +87,11 @@ Sr Staff Engineer
 </td></tr>
 <tr><td markdown="1" style="font-weight:bold">
 
-- Technical expert in all key components and solid on the entire codebase.
+- Technical expert in most key components and solid on 2 products.
```
Comment (partner0) [inline eng-paths/sr_staff_engineer.md:90]: I don't like that change, I think the bar should be the entire codebase. Ricardo might be an exception to the rule given his history in devops over code, and Michael may be a little rusty in React, but h makes up for it being the Django authority in staff meetings. I think the bar should be: Solid on the entire codebase.

File: eng-paths/staff_engineer.md [hunk 15]
```diff
@@ -88,14 +89,16 @@ Whole platform and organization
...
 <tr><td>
 
-Sr Software Engineer 2
+Sr Software Engineer 3
 
 </td></tr>
 <tr><td  markdown="1">
 
-- Technical expert on 2 products / key components, solid on the rest of the platform.
+- Technical expert on multiple products / key components, knowing all the extent of their core team's domain.
```
Comment (partner0) [inline eng-paths/staff_engineer.md:97]: Same as above, staffs need to be solid on the entire platform, and don't have a core team anymore...

File: eng-paths/staff_engineer.md [hunk 16]
```diff
@@ -117,9 +122,11 @@ Sr Staff Engineer
 </td></tr>
 <tr><td  markdown="1">
 
-- Technical expert in all key components and solid on the entire codebase.
+- Technical expert in most key components and solid on 2 products.
```
Comment (partner0) [inline eng-paths/staff_engineer.md:125]: Same as above again

File: eng-paths/director_of_engineering.md
Comment (justipaula) [reply eng-paths/director_of_engineering.md:1]: I see it as Senior Staff in mine 😮 you're talking about the eng-paths.md, right? there was a typo but it was fixed last Friday morning. Could you triple check if it's still appearing for you as Staff and let me know? 🙏🏽 
image

File: eng-paths/sr_staff_engineer.md (hunk 14 above)
Comment (justipaula) [reply eng-paths/sr_staff_engineer.md:90]: omg totally, thank you for flagging @partner0, this wasn't intentional but rather a copy action done at the wrong line 🤦🏽‍♀️ this is currently duplicated on the sr staff and SE3, but belongs only to SE3. I'll roll it back, 100% aligned, apologies + thanks for catching it!

File: eng-paths/staff_engineer.md (hunk 15 above)
Comment (justipaula) [reply eng-paths/staff_engineer.md:97]: ohh, that's not the staff one, that's the SE3 description appearing inside the staff file given the way we compile/show the data. Are we good here then? 🙂
